src/figuresClasses.ts

```diff
@@ -1,11 +1,75 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea: () => number;
+}
+
+function areaRound(area: number): number {
+  return Math.floor(area * 100) / 100;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+    public shape: string = `triangle`,
+  ) {
+    const sides = [a, b, c].sort((side1, side2) => side1 - side2);
+    const isValidSides = sides[2] < sides[0] + sides[1];
+
+    if (a <= 0 || b <= 0 || c <= 0 || !isValidSides) {
+      throw new Error('any side must be greater than 0');
```

Comment:
The error message thrown here is not descriptive enough. It should explain what is wrong with the sides of the triangle. For example, 'Triangle sides must be positive and satisfy the triangle inequality'.